core/container/src/main/java/org/wildfly/swarm/internal/MavenArgsParser.java

```diff
@@ -0,0 +1,54 @@
+package org.wildfly.swarm.internal;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.StringTokenizer;
+
+/**
+ * Created by hbraun on 22.08.17.
+ */
+public class MavenArgsParser {
```

Comment:
I'll add support for `--file` but would ignore the `-fanotherpom.xml` case for now. This is mostly about us developing and verifying the boosters